app/assess/models/application.py

```diff
@@ -49,3 +49,11 @@ def get_question(self, index: int):
         if self.questions:
             return self.questions[index]
         return None
+
+    def get_question_field(self, page_title):
+        fields = []
+        for question in self.questions:
+            if page_title == question.title:
```

Comment:
Better style is to have the constant on the right hand side
```python
if question.title == page_title:
```

Reply:
Done. Thank you! :)